src/popup/components/MultisigProposalConsensus.vue

```diff
@@ -2,9 +2,17 @@
   <div class="transaction-multisig-consensus">
     <div
       v-if="!isPendingMultisigTxCompleted"
-      class="label"
+      class="consensus-label"
     >
-      <span> {{ $t('multisig.consensus') }} </span>
+      <div class="label">
+        <span> {{ $t("multisig.consensus") }} </span>
+        <BtnHelp
+          class="btn-help"
+          is-custom-emit
+          @help="openModal(MODAL_CONSENSUS_INFO)"
```

Comment:
I suggest to create a custom method for this action to have a better TS support/validation.